pkg/upstream/cluster/loadbalancer.go

```diff
@@ -45,6 +46,7 @@ func init() {
 	}
 	RegisterLBType(types.RoundRobin, rrFactory.newRoundRobinLoadBalancer)
 	RegisterLBType(types.Random, newRandomLoadBalancer)
```

Comment:
it should update [convertLbPolicy]( https://github.com/mosn/mosn/blob/master/pkg/xds/conv/convertxds.go#L985) function  for Istio as a configuration source scenario.